### ISSUE TITLE
Unable to use app engine Modules with NoseGAE

### ISSUE THREAD

**Issue by robwithhair** [Trii/NoseGAE#22]
Hi there,

I've been trying to use more than one app engine module with NoseGAE but it doesn't seem possible.  when I run `dev_appserver.py api-v1/ php-ancillary-services.yaml` I am specifying firstly my project root and then my additional module, however I can't seem to find a way to do this in my functional tests?  The error I am getting is..
```
  File "/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/api/modules/modules.py", line 146, in _CheckAsyncResult
    raise mapped_error()
InvalidModuleError
```
I assume this is because noseGAE has no idea about my additional modules, but how would I let NoseGAE know about my additional modules?  I tried changing some of the source for NoseGAE but I can't get it working.  I don't think this'll be something I can get working myself so would appreciate any help.

**Comment by Trii** [comment]
I will take a look at this today as well. Can you give me the command you are using to call nosetests?

**Comment by sadovnychyi** [comment]
`handlers` section is not being used by nosegae at all, you test your code bypassing the yaml files. Only queue.yaml is properly used.

**Comment by Trii** [comment]
@robwithhair this is the WIP for modules usage: https://github.com/Trii/NoseGAE/pull/26

After I put together a few more tests I will make a release.

**Comment by Trii** [comment]
Fixed in #24
